docs/basics/k8s_configure.md

```diff
@@ -0,0 +1,132 @@
+              url: https://my-host:8443/admin/metrics.json
+``` 
+
+> To get a quick, first picture of a Flex configuration file, you can start following our [basic, step-by-step tutorial](../../basic-tutorial.md) or check existing config files under [/examples](../../examples).
+
+## <a name='Linktoaseparateconfigurationfile'></a>Link to a separate configuration file
+
+You can store the Flex configuration in a separate YAML file (for example, after [developing and testing a config file](../development.md)) and reference it by replacing `config` with `config_template_path` property, which contains the path of the Flex config file.
+
+This way, the equivalent of previous configMap from the previous section would contain:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-integration-cfg # aimed to be safely overridden by users
+  namespace: default
+data:
+  nri-flex.yml: |
+    integrations:
+      - name: nri-flex
+        config_template_path: /path/to/flex/integration.yml # Reference to a separate Flex config file
+``` 
+While this would work, you would either have to create a new configMap with the contents of the external file and mount it into the container, 
+or you would have to modify the original container image to include the file. For this reason we recommend you embedded you configuration as explained in the previous section. 
+
+## <a name='#Addmultipleflexconfigurations'></a>Add multiple Flex configurations
+
+You can also add multiple Flex configurations to the configMap. There are two ways to do this:
+
+- add multiple Flex entries to the same "file"
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-integration-cfg # aimed to be safely overridden by users
+  namespace: default
+data:
+  nri-flex.yml: |
+    integrations:
+      - name: nri-flex-1
+        config:
+          name: example
+          apis:
+            - event_type: ExampleSample
+              url: https://my-host:8443/admin/metrics.json
+      - name: nri-flex-2
+        config:
+          name: other-example
+          apis:
+            - event_type: OtherExampleSample
+              url: https://my-other-host:8080/metrics.json
+```
+
+- add distinct configMap entries, one per "file"
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-integration-cfg # aimed to be safely overridden by users
+  namespace: default
+data:
+  nri-flex-1.yml: |
+    integrations:
+      - name: nri-flex-1
+        config:
+          name: example
+          apis:
+            - event_type: ExampleSample
+              url: https://my-host:8443/admin/metrics.json
+  nri-flex-2.yml: |
+    integrations:
+      - name: nri-flex-2
+        config:
+          name: other-example
+          apis:
+            - event_type: OtherExampleSample
+              url: https://my-other-host:8080/metrics.json
+```
+
+Which one you use is up to you. There are no distinctive advantages of using one over the other.
```

Comment:
I thought running multiple in the same "file" would run sync, and splitting them out would allow async, is this not correct?

Reply:
This case is not the same as running multiple apis in the same Flex config file. 
Is this case the agent will  still run multiple independent instances of Flex so they run in parallel. 
This case is just for "organization" purposes, has no impact in the way Flex runs